### DISCUSSION TITLE
~ is not in the layout in ZMK with Eurokey layout Macos

### DISCUSSION THREAD

**Discussion by gogamid** [manna-harbour/miryoku#298] [Q&A] [answered]
![image](https://github.com/manna-harbour/miryoku/assets/36050790/16a300be-272e-4606-9f9a-8ba2f7f74962)
this is the layout i compiled, but sym+pinkie or numb+pinke give § or with shift ± symbols, instead of `and ˜

**Comment by gogamid** [comment]
oh i had to change my keyboad type to Ansi in macos settings

**Comment by manna-harbour** [comment, marked as answer]
See https://github.com/manna-harbour/miryoku/discussions/197#discussioncomment-5240943.
